NationStatesAPIBot/NationStatesAPIBot/Services/DumpDataService.cs

```diff
@@ -1,6 +1,245 @@
-﻿namespace NationStatesAPIBot.Services
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+using DumpData;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NationStatesAPIBot.Types;
+
+namespace NationStatesAPIBot.Services
 {
     class DumpDataService
     {
+        private readonly BaseApiService _apiService;
+        private static List<NATION> _nations;
+        private static List<REGION> _regions;
+        private static Task _updating;
+
+        public DumpDataService(ILogger<RecruitmentService> logger, IOptions<AppSettings> appSettings, BaseApiService apiService, CancellationToken cancel)
```

Comment:
You aren't using appSettings and logger are you?

Reply:
I'll remove the unused parameters.